src/core/EdFi.DataManagementService.Core.Tests.Unit/Middleware/DuplicatePropertiesMiddlewareTest.cs

```diff
@@ -265,12 +259,8 @@ public void It_returns_message_body_with_failure_duplicated_property()
             _context
                 .FrontendResponse.Body?.ToJsonString()
                 .Should()
-                .Contain("Data Validation Failed");
-
-            _context
-                ?.FrontendResponse.Body?.ToJsonString()
-                .Should()
-                .Contain(
+                .ContainAll(
+                    "Data Validation Failed",
```

Comment:
I found this interesting method that let's you look for multiple strings in the result, instead of having to write two different asserts.